packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -123,27 +123,6 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
         gltf,
         gltfEmissiveTexture ? gltfEmissiveTexture : null,
     );
-
-    const message = (textureType: string) => {
-      console.info(`A group of three.js materials are represented as a
-        single material but share different ${textureType} textures.`);
```

Comment:
These logs were being recorded for the variant example, which indicates to me this is not really an error. I went ahead and removed them for now to keep from polluting the console.